build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation 'com.fasterxml.jackson.core:jackson-annotations:2.15.2'
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.2'
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.15.2'
+    implementation "com.github.spotbugs:spotbugs:4.8.3"
```

Comment:
Line 67 refers to 4.7.3, and `exclude.yaml` refers to 4.6.0. Can we address these disparities?

Also, the line for the plugin (`id "com.github.spotbugs" version "5.0.14"`) has yet another version. I'm not sure if this is specific to the plugin though, which might be ok.

Finally, does it really make sense to add this as a _dependency_ for our project? Seems much more like a development tool that's nice-to-have.

Reply:
1. Fixed the version discrepancy
2. The version 5.0.14 is the plugin version, which contains a different spotbugs version
3. I can't recall exactly why this was required, but it would fail if I marked it as `testImplementation` instead of `implementation`. iirc, since it's used in the build process, it has to be included in the jpackage. I can try to re-investigate what happens if I change it back and better document it if needed.

Reply:
Nice, thanks. What about this one?

> exclude.yaml refers to 4.6.0. Can we address these disparities?

Re: 3. maybe yezr@ or you could file an issue for that?

Reply:
Ah my bad, glossed over that! Fixed